torchmdnet/scripts/train.py

```diff
@@ -130,6 +131,7 @@ def get_args():
     if args.inference_batch_size is None:
         args.inference_batch_size = args.batch_size
 
+    os.makedirs(os.path.abspath(args.log_dir), exist_ok=True)
```

Comment:
Creates the log folder instead of getting an error in save_argparse

Reply:
Damn thanks for this. I was really tired of having to create the folder.

Reply:
100%